### ISSUE TITLE
Status update?

### ISSUE THREAD

**Issue by pepijndevos** [OpenHAK/OpenHAK_Hardware#1]
Hey there! I found this repo via a long journey via Google, Hackaday, Kickstarter, etc.
I was wondering what the status of this is? Obviously the kickstarter failed, and your MCU is no longer in production, but I'm curious how far along you got, and how well everything works?

I'm particularly curious how well the BMI160 and MAX30101 work for counting steps and heart rate, and how you solved power management.

**Comment by biomurph** [comment]
@pepijndevos 
Thanks for taking the long journey!
This past summer, we made a short production run (100 units) to make OpenHAK as the badge for the Biohacking Village at DEF CON. The system works pretty darn good. BMI160 has built in tools for getting steps, and the MAX heart rate sensor works well too. The wrist is not the most ideal place to get heart rate, and it's known that PPG is not ideal over skin with lots of melanin...

The firmware will connect to your phone, and then wake up every 10 minutes to try and get heart rate and send data to the phone if it's still connected. It will keep persistent data for 3 days (storage and battery life).

We have a couple of BLE modules that we're looking at to push the project further, but that will also take time and effort ;)
Look for updates in 2020

**Comment by pepijndevos** [comment]
Glad to hear the project is not dead yet, and looking forward to updates in the coming year. I suppose I can wait it out with my Fitbit ;)

Being a bit of a tinfoil hat guy, I am personally not really interested in BLE, and would rather sync over USB to my laptop. BLE is only useful if you want to do notifications and control music and stuff like that IMO. I suspect I'm a minority here. Luckily it's open source and I can modify the firmware ;)

I wonder if you have considered using Gadgetbridge rather than writing your own app?

Has anyone tried doing sleep tracking? This is my primary interest for a tracker, but seems challenging to do well. From what I hear, the sleep tracking in Gadgetbridge is not that good at all. Maybe I should export my Fitbit data and see what I can do.

**Comment by biomurph** [comment]
I think some of the radios we are looking at have USB... Also, if you do any work on sleep tracking, can you create a repo here ore elsewhere? Gitlab is maybe more useful for that? It would be great to start a conversation about best ways to do that. It's also in my interest

**Comment by dv29** [comment]
Is this still alive, I'm interested in sleep tracking as well. Was there an alternative found for BLE?

**Comment by biomurph** [comment]
Depends on what you mean by 'alive'.

No development has happened on any BLE module upgrade. 

Likely candidates for BLE alternative is an nRF l52840 module.

Please check out https://www.emotibit.com/ their tech rocks.

**Comment by dv29** [comment]
Emotibit is how i stumbled upon this. I could not find their specs or open source projects or anything. Thanks though. I'll try it out.

**Comment by biomurph** [comment]
Yeah, Connected Future Lab is a good friend of mine. I have had access to their prototype Emotibit and it is a pleasure to work with. You would get more data from it for you sleep project, and it's all done! Nice GUI interface. SD card storage and even streaming.
